cugraph/ConnectedComponents.ipynb

```diff
@@ -0,0 +1,443 @@
+      "   labels  vertices\n",
+      "0       1         0\n",
+      "1       1         1\n",
+      "946       1       946\n",
+      "1084       1      1084\n"
+     ]
+    }
+   ],
+   "source": [
+    "expr = \"labels == 1\"\n",
+    "component = df.query(expr)\n",
+    "print(\"Vertex Ids that belong to component label 1 : \")\n",
+    "print(component)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3b. Call Strongly Connected Components"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "    labels  vertices\n",
+      "0       0         0\n",
+      "1       0         1\n",
+      "2       2         2\n",
+      "3       2         3\n",
+      "4       2         4\n",
+      "5       2         5\n",
+      "6       2         6\n",
+      "7       7         7\n",
+      "8       7         8\n",
+      "9       7         9\n",
+      "[1579 more rows]\n"
+     ]
+    }
+   ],
+   "source": [
+    "df = cugraph.strongly_connected_components(G)\n",
+    "print(df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Get total number of strongly connected components"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "label_gby = df.groupby('labels')\n",
+    "label_count = label_gby.count()\n",
+    "largest_component = label_count.nlargest(n = 1, columns = 'vertices')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total number of components found :  396\n",
+      "Size of the largest component is found to be :  379\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Total number of components found : \", len(label_count))\n",
+    "print(\"Size of the largest component is found to be : \", largest_component['vertices'][0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Output vertex ids belonging to a strongly connected component label"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Vertex Ids that belong to component label 2 : \n",
+      "   labels  vertices\n",
+      "2       2         2\n",
+      "3       2         3\n",
+      "4       2         4\n",
+      "5       2         5\n",
+      "6       2         6\n"
+     ]
+    }
+   ],
+   "source": [
+    "expr = \"labels == 2\"\n",
+    "component = df.query(expr)\n",
+    "print(\"Vertex Ids that belong to component label 2 : \")\n",
+    "print(component)"
+   ]
+  },
```

Comment:
It would be ideal to have a story around this notebook or conclude on why this result is useful.

Reply:
Done